cli/cmd/upgrade.go

```diff
@@ -141,6 +141,9 @@ func (options *upgradeOptions) validateAndBuild(k kubernetes.Interface, flags *p
 	if options.proxyAutoInject {
 		configs.GetGlobal().AutoInjectContext = &pb.AutoInjectContext{}
 	}
+	if options.controlPlaneVersion != "" {
+		configs.Global.Version = options.controlPlaneVersion
```

Comment:
Mind using `GetGlobal()` here to remain consistent with the other parts of this method?